samples/subsys/fs/nffs/README.rst

```diff
@@ -0,0 +1,35 @@
+.. _nrf52_NFFS_sample:
+
+nrf52 NFFS Read/Write Sample
+############################
+
+This is a simple application demonstrating how to read & write persistence data on 
+flash of the nRF52 SoC, using NFFS file system.
+
+Requirements
+************
+
+This sample has been tested on the Nordic nRF52840-PDK board, but would
+likely also run on the nrf52_pca10040 board.
+
+Building and Running
+********************
+
+This sample can be found under ``$(ZEPHYR_BASE)/samples/boards/nrf52/nffs`` in the
+Zephyr tree.
+
+This example adds the ``$(ZEPHYR_BASE)/ext/fs/nffs`` include directory to the project
+by providing its own ``CMakeLists.txt`` within the sample directory.
+
+Use zephyr-app-commands to build the application. See examples of usage in ``doc/getting_started``
```

Comment:
Not quite right.  The ``zephyr-app-commands`` directive is a documentation macro used to create consistent-looking documentation for samples.  You previously had a set of command line steps in an earlier version of this document.  Instead of using those explicit steps.  Delete this line (24) and use the zephyr-app-commands directive here, with  something like this:

    .. zephyr-app-commands::
       :zephyr-app: samples/boards/nrf52/nffs
       :board: nrf52840_pca10056
       :goals: build flash
       :compact:

When the documentation is generated this will expand into a set of commands looking like this:

    # On Linux/macOS
    cd $ZEPHYR_BASE/samples/boards/nrf52/nffs
    mkdir build && cd build

    # On Windows
    cd %ZEPHYR_BASE%\samples\boards\nrf52\nffs
    mkdir build & cd build

    cmake -GNinja -DBOARD=nrf52840_pca10056 ..
    ninja
    ninja flash